deepdives/02-middleware/example/middleware.js

```diff
@@ -0,0 +1,46 @@
+/*
+  Example Authentication Middleware.
+  This is meant to serve as a simple example of how
+  some middleware can be structured.
+  By no means should this middleware ever be used
```

Comment:
No error handling, doesn't really do much of anything, passport is a better alternative.

Maybe I'll just recommend Passport as an alternative.